src/test/regress/spec/isolation_citus_pause_node.spec

```
@@ -0,0 +1,219 @@
+setup
+{
+	SET citus.shard_replication_factor to 1;
+
+	create table city	(id int , name text );
+	SELECT create_reference_table('city');
+
+    CREATE TABLE company(id int primary key, name text, city_id int);
+	select create_distributed_table('company', 'id');
+
+	create table employee(id int , name text, company_id int  );
+	alter table employee add constraint employee_pkey primary key (id,company_id);
+
+	select create_distributed_table('employee', 'company_id');
+
+	insert into city values(1,'city1');
+	insert into city values(2,'city2');
+
+
+	insert into company values(1,'c1', 1);
+	insert into company values(2,'c2',2);
+	insert into company values(3,'c3',1);
+
+	insert into employee values(1,'e1',1);
+	insert into employee values(2,'e2',1);
+	insert into employee values(3,'e3',1);
+
+	insert into employee values(4,'e4',2);
+	insert into employee values(5,'e5',2);
+	insert into employee values(6,'e6',2);
+
+	insert into employee values(7,'e7',3);
+	insert into employee values(8,'e8',3);
+	insert into employee values(9,'e9',3);
+	insert into employee values(10,'e10',3);
+
+
+}
+
+teardown
+{
+    DROP TABLE employee,company,city;
+}
+
+session "s1"
+
+step "s1-begin"
+{
+    BEGIN;
+}
+
+step "s1-node-not-found"
+{
+	DO $$
+	DECLARE
+		v_node_id int:= -1;
+		v_node_exists boolean := true;
+		v_exception_message text;
+		v_expected_exception_message text := '';
+	BEGIN
+		select nextval('pg_dist_node_nodeid_seq')::int into v_node_id;
+		select citus_pause_node_within_txn(v_node_id) ;
+	EXCEPTION
+		WHEN  SQLSTATE 'P0002' THEN
+			GET STACKED DIAGNOSTICS v_exception_message = MESSAGE_TEXT;
+			v_expected_exception_message := 'node ' || v_node_id || ' not found';
+			if v_exception_message = v_expected_exception_message then
+				RAISE NOTICE 'Node not found.';
+			end if;
```

What happens when you don't transform the ERROR to a NOTICE like this:
```suggestion
	BEGIN
		select nextval('pg_dist_node_nodeid_seq')::int into v_node_id;
		select citus_pause_node_within_txn(v_node_id) ;
```

I think we'll see the error in the test output, which would be a fine way to test, and reduced the testing code.

---

Node id is a changing value and error message includes node id. Therefore, I'm adding this exception handling to fix the error message